azure-pipelines.yml

```diff
@@ -79,6 +79,7 @@ steps:
   failOnStderr: true
 
 - bash: |
+    export source_branch=$(System.PullRequest.SourceBranch)
```

Comment:
I would recommend pass the `source_branch` as argument to Makefile instead of using environment variable.

Reply:
OK. Now the `source_branch` is passed as argument.